webapp/src/js/app.js

```diff
@@ -140,7 +140,7 @@ const createReduxLoggerConfig = Selectors => ({
     $translateProvider.useSanitizeValueStrategy('escape');
     $translateProvider.addInterpolation('TranslationMessageFormatInterpolation');
     $translateProvider.addInterpolation('TranslationNullInterpolation');
-    $translateProvider.useMissingTranslationHandlerLog();
+    //$translateProvider.useMissingTranslationHandlerLog();
```

Comment:
Ping.

Reply:
Oops :-s reverted. 
Side note: do you think maybe we should update this so it's disabled in a dev environment. 
I find myself always commenting this out when doing webapp dev work cause missing translations just flood my console.

Reply:
Yes that'd be fine. The most important time for this to show is during configuration so as long as it's just hidden from core dev then no worries.

I assume it's also unhelpful in production because nobody checks - the ideal solution would collate all the missing keys and submit them to the server somewhere rather than just dumping them out in the console.